command/command_testonly/server_testonly_test.go

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+//go:build testonly
+
+package command_testonly
```

Comment:
in order to make sure these tests actually run, we'll need to add the command directory here: https://github.com/hashicorp/vault/blob/main/.github/workflows/test-go.yml#L159